### ISSUE TITLE
Twiggy 4.3T ROMs

### ISSUE THREAD

**Issue by Lucas-mother3** [minivmac/minivmac#1]
I would like to ask for adding support for a different version of the ROM using slowmacuser's dumps on Macintosh Garden. The ROM wouldn't work normally unless if you disabled ROM checksum by compile time. I used stable 36.04, I would like to add support for it and many Twiggy prototypes in the future including one which had an even earlier version, if a dump of it was released on MacGUI.

Here's some info:
```
Filename: Twig43.ROM
MD5 hash: e4faf3ecb169b875a1e66abbd5306b52
CRC32 hash: d2c42f18
How to replicate:
1. Go and download "TwiggyMacROMsCombined.zip" on Macintosh Garden 
2. Compile the interlace program
2.1. Clone this repo: https://github.com/drojaazu/interleave
2.2. Install clang
2.3. Compile using make
3. Copy the ROMs in the same directory as the program
4. Copy this command: ./interleave HIGHX.BIN LOWX.BIN > Twig43.ROM
5. Verify the results by checking the string "ROM 4.3T 07/04/83" on a hex editor.
```

**Comment by egrath** [comment]
Thanks for the report, will take a look into it!

**Comment by egrath** [comment]
Can you please test out the branch test01 which includes the checksum for the above ROM. Works here without any issues (boots the patched images), but want a second opinion before merging it onto the master branch.

![2022-05-27 22_14_24-Mini vMac](https://user-images.githubusercontent.com/1955895/170783463-b987c7d0-82ac-491b-9697-13f88b2a8f27.png)

**Comment by Lucas-mother3** [comment]
Pretty good results actually! The build works, I made a custom build again, which includes credits. Keep maintaining this until Paul came back. I wish he’s fine, touching grass and doing normal ***@***.***
***@***.***

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________

**Comment by Lucas-mother3** [comment]
I actually sent it via Mail for Windows, so check the attachments in the email itself.

**Comment by egrath** [comment]
Solved with b01f97088e2efa86835e77e1f595788bf220ea04

**Comment by Lucas-mother3** [comment]
Surely! I do have a repo that host 36.04 binaries, so I would like to!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
